library/user.go

```diff
@@ -8,12 +8,13 @@ import "fmt"
 
 // User is the library representation of a user.
 type User struct {
-	ID     *int64  `json:"id,omitempty"`
-	Name   *string `json:"name,omitempty"`
-	Token  *string `json:"token,omitempty"`
-	Hash   *string `json:"-"`
-	Active *bool   `json:"active,omitempty"`
-	Admin  *bool   `json:"admin,omitempty"`
+	ID        *int64  `json:"id,omitempty"`
+	Name      *string `json:"name,omitempty"`
+	Token     *string `json:"token,omitempty"`
+	Hash      *string `json:"-"`
+	Favorites *string `json:"favorites,omitempty"`
```

Comment:
🤔 I don't think we want this to be a single string. I think we want it to be `*[]string`.

This follows the pattern we have for the secrets `library.Secret.Images` and `library.Secret.Events` fields.

https://github.com/go-vela/types/blob/master/library/secret.go#L24-L25

Reply:
Oh I misunderstood, I was thinking string that contains `json` string that could be unmarshaled to a slice of strings. Good call! updating

Reply:
updated!